### PR TITLE
Doc board assembly

### PR DESCRIPTION
**Pull request by KarlSchu**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,11 +17,14 @@ extras/automation/geckodriver.exe
 extras/automation/config.yaml
 src/sdkconfig.h
 sdkconfig.esp32dev
+docs/electroCad/fp-info-cache
+docs/electroCad/SandBox.sch-bak
+docs/electroCad/ser4.bck
+docs/electroCad/ser4.sch-bak
+docs/electroCad/ser4.kicad_pcb-bak
+docs/electroCad/MainComputer.sch-bak
+docs/electroCad/SteeringWheel.sch-bak
```
Comment (andreaskuster) [inline .gitignore:26]: Couldn't we ignore all files i.e. using `*` operator?

File: README.md [hunk 2]
```diff
@@ -15,27 +15,26 @@
...
+| ESP32:5      | SPI:CLK                |
+| ESP32:12     | 1Wire                  |
+| **ESP32:13** | GPIO:LED_BUILTIN       |
+| ESP32:33     | GPIO:PCF8574_INTERRUPT |
+| **ESP32:14** | SPI:CS_SD_CARD         |
+| ESP32:36     | SPI:CS_TFT_SCREEN      |
+| ESP32:4      | SPI:DC                 |
+| ESP32:21     | SPI:RST                |
+| ESP32:25     | GPIO:INTERRUPT         |
+| ESP32:25 DAC | CAN:TX                 |
```
Comment (andreaskuster) [inline README.md:36]: Is this conflict fixed?

File: datasheets/254_eng_tds--SdCard.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# MicroSD card breakout board+
+
+[Technical Details (Adafruit)](https://www.adafruit.com/product/254#technical-details)
+
+![image-20210829205636512](.254_eng_tds--SdCard/image-20210829205636512.png)
+
+![MicroSD card breakout board+](.254_eng_tds--SdCard/254-05.jpg)
+
+![additional product photo](.254_eng_tds--SdCard/254-03.jpg)
+
```
Comment (andreaskuster) [inline datasheets/254_eng_tds--SdCard.md:10]: Please apply meaningful names to the images.

File: docs/ADS1115--ADC0_ADC1_ADC2-infos.md [hunk 4]
```diff
@@ -23,6 +23,12 @@ Usage in car hull:
 
 ![img](.ADS1115--ADC0_ADC1_ADC2-infos/FSR-CurcuitDiagram.drawio.png)
 
+### Board Assembly
+
+![image-20210823130410331](.ADS1115--ADC0_ADC1_ADC2-infos/image-20210823130410331.png)
+
+![image-20210823130639854](.ADS1115--ADC0_ADC1_ADC2-infos/image-20210823130639854.png)
+
```
Comment (andreaskuster) [inline docs/ADS1115--ADC0_ADC1_ADC2-infos.md:31]: Please rename and add them to a non-hidden folder.

File: docs/DOCU--ILI934-PCF8574-ESP32-Hazzy.md [hunk 5]
```diff
@@ -82,3 +82,12 @@ It was measured at full screen black and full screen white (without any addition
...
 ![image-20210327121949117](.DOCU--ILI934-PCF8574-ESP32-Hazzy.assets/image-20210327121949117.png)
+
+## Mechanical Construction
+
+![2021-08-23 20.14.35](.DOCU--ILI934-PCF8574-ESP32-Hazzy/2021-08-23 20.14.35.jpg)
+
+
+
+![Untitled Diagram.drawio](.DOCU--ILI934-PCF8574-ESP32-Hazzy/mechanicalDimensions.drawio.png)
+
```
Comment (andreaskuster) [inline docs/DOCU--ILI934-PCF8574-ESP32-Hazzy.md:93]: Please rename and add them to a non-hidden folder.

File: docs/PCF8574--IOExt0_IOExt1_IOExt2-infos.md [hunk 6]
```diff
@@ -28,3 +28,8 @@ Usage in car hull:
 - GND, VCC, I2C (SDA,SCL), Interrupt
 
 ![img](.PCF8574--IOExt0_IOExt1_IOExt2-infos/IO-Ext-CurcuitDiagram.drawio.png)
+
+### Board Assembly
+
+![image-20210823131457974](.PCF8574--IOExt0_IOExt1_IOExt2-infos/image-20210823131457974.png)
+
```
Comment (andreaskuster) [inline docs/PCF8574--IOExt0_IOExt1_IOExt2-infos.md:35]: Please rename and add them to a non-hidden folder.

**Review by andreaskuster — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (KarlSchu) [reply .gitignore:26]: removed it completely --> EasyEda-CAD

File: datasheets/254_eng_tds--SdCard.md (hunk 3 above)
Comment (KarlSchu) [reply datasheets/254_eng_tds--SdCard.md:10]: done

File: docs/ADS1115--ADC0_ADC1_ADC2-infos.md (hunk 4 above)
Comment (KarlSchu) [reply docs/ADS1115--ADC0_ADC1_ADC2-infos.md]: done

File: docs/PCF8574--IOExt0_IOExt1_IOExt2-infos.md (hunk 6 above)
Comment (KarlSchu) [reply docs/PCF8574--IOExt0_IOExt1_IOExt2-infos.md]: done

File: docs/DOCU--ILI934-PCF8574-ESP32-Hazzy.md (hunk 5 above)
Comment (KarlSchu) [reply docs/DOCU--ILI934-PCF8574-ESP32-Hazzy.md]: done

File: README.md (hunk 2 above)
Comment (KarlSchu) [reply README.md]: done

**Pull request merged**